src/index.js

```diff
@@ -27,6 +27,11 @@ export function select(name, options, value) {
   return manager.knob(name, { type: 'select', options, value });
 }
 
+export function date(name, value) {
+  const r =  manager.knob(name, { type: "date", value: value.toISOString() });
```

Comment:
I like to use a timestamp here. It's more generic in JS and smaller.
I think you need to change several places.